ckanext/datitrentinoit/model/mapping.py

```diff
@@ -269,3 +269,26 @@ def _extras_as_dict(extras):
             extras_as_dict.append({'key': key, 'value': value})
 
     return extras_as_dict
+
+
+def create_opencity_package_dict(harvest_object, config):
+    json_dict = json.loads(harvest_object.content)
+
+    package_dict = {
+        "title": json_dict["title"],
+        "short_description": json_dict["short_description"],
```

Comment:
CKAN does not have a "short_description" field. 
You already worked on this field in the statsweb harvester.

Reply:
Okay @etj 
I thought I need to include that because it was in the description, misunderstood.
![image](https://user-images.githubusercontent.com/104504055/168800104-f9641315-19ad-4685-80f0-83aa16b690c3.png)

Reply:
Done